README.md

```diff
@@ -159,7 +159,7 @@ contains FHIR profiles, example resources, and implementation guides for
 a specific version of FHIR. See https://simplifier.net/learn.
 
 Everything we put in this folder will eventually be published to the corresponding
-[KidsFirstSTU3 Simplifier Project](https://simplifier.net/kidsfirststu3) so that
+[KidsFirstR4 Simplifier Project](https://simplifier.net/kidsfirstr4) so that
```

Comment:
The case difference between the URL project name and the SIMPLIFIER_PROJECT_NAME variable probably doesn't matter.

Reply:
Correct, case doesn't matter